lib/LaTeXML/Core/Mouth.pm

```diff
@@ -201,16 +201,9 @@ sub handle_escape {    # Read control sequence
   if ((defined $cc) && ($cc == CC_LETTER)) {    # For letter, read more letters for csname.
     while ((($ch, $cc) = getNextChar($self)) && $ch && ($cc == CC_LETTER)) {
       $cs .= $ch; }
+    # We WILL skip spaces, but not till next token is read (in case catcode changes!!!!)
+    $$self{skipping_spaces} = 1;
```

Comment:
Ah, crucial difference, nicely spotted.

Reply:
Question: is this a Mouth attribute or a Gullet attribute? Could there be a situation where one mouth sets the skipping spaces mode, but they go on skipping into a second mouth?

Reply:
In TeX, it's a global variable. But given how it took so long to notice that the spaces hadn't yet been skipped, I'm figuring the more hidden it is, the better.